src/main/java/com/example/gsheets/controller/SheetController.java

```diff
@@ -1,24 +1,35 @@
 package com.example.gsheets.controller;
 
+import com.example.gsheets.controller.dto.StudentDTOController;
+import com.example.gsheets.service.SheetService;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
 
-@RestController
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import static  com.example.gsheets.controller.mapper.ControllerDTOMapper.*;
+import static com.example.gsheets.service.mapper.StudentMapper.fromDTOControllerToStudent;
 
+@RestController
 public class SheetController {
+    SheetService service = new SheetService();
     @GetMapping("/sheet/{id}")
```

Comment:
are you getting "sheet" or "student"? should be /student/{id}

Reply:
done